http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1338,8 +1305,8 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       _ <- wsmResource.resourceType match {
         case WsmResourceType.AzureManagedIdentity =>
           F.blocking(wsmApi.deleteAzureManagedIdentity(workspaceId.value, wsmResource.resourceId.value))
-        case WsmResourceType.AzureDatabase =>
-          deleteAzureDatabaseResource(workspaceId, app, wsmResource)
+        case WsmResourceType.AzureDatabase | WsmResourceType.AzureKubernetesNamespace =>
```

Comment:
Would it make sense to include the deletion of the managed identity in the `deleteAndPollWsmNamespaceResource` pending renaming of the function per my comment above?

Reply:
+1

Reply:
I thought about that, but it makes things a bit complicated because the `managedIdentity` is the only resource we don't poll on, and so we don't need the majority of that function. I condensed the polling WSM resources in the same function to avoid repeated code when generating the jobId, delete request etc, but that has the byproduct of some redundant case statements that you guys pointed out

Reply:
I can try to experiment a bit more with it though

Reply:
decided to pull some of the setup out of the shared function and have a different polling function for each resource, much cleaner this way